ContactDetailsApi/V2/Gateways/TenureDbGateway.cs

```diff
@@ -42,10 +40,22 @@ public async Task<PagedResult<TenureInformation>> ScanTenures(string paginationT
         var resultsSet = await search.GetNextSetAsync().ConfigureAwait(false);
         paginationToken = search.PaginationToken;
 
-        // _logger.LogInformation("Returned {resultsCount} results from IDynamoDBContext.ScanAsync for TenureInformationDb with pagination token {paginationToken}", resultsSet.Count, paginationToken);
+        TenureInformation SafeToDomain(TenureInformationDb tenure)
+        {
+            try
+            {
+                return tenure?.ToDomain();
+            }
+            catch (Exception e)
+            {
+                _logger.LogError(e, "Error: Failed to convert tenure {Tenure} to TenureInformation domain", tenure?.Id);
+                return null;
+            };
+        };
```

Comment:
Never seen stuff like that before.

Could you tell more about what problem this is solving?

Either that or add it to the PR Description to get this documented.
If there's any dirty data causing the issues, it would be good for this to be noted down.

Reply:
It's so that we can identify if there are any issues with the data when applying the data conversions